tests/integration/pragma/test_pragma_stmts.rs

```diff
@@ -7,6 +7,19 @@ mod tests {
     use rexpect::session::{spawn_command, PtySession};
     use std::process;
 
+    #[test]
+    fn test_pragma_page_count() -> Result<(), Error> {
+        let mut child = spawn_command(run_cli(), Some(1000))?;
+        child.exp_regex("limbo>")?; // skip everything until limbo cursor appear
+        child.exp_regex(".?")?;
+        child.send_line("pragma page_count;")?;
+        child.exp_string("0")?;
+        child.exp_regex("limbo>")?;
+        child.send_line("create table foo(bar); pragma page_count;")?;
+        child.exp_string("2")?;
+        quit(&mut child)
+    }
+
```

Comment:
What is this wizardy. Can we just have a TCL test? You should be able to add the equivalent of this test in `testing/pragma.test`.

Something like:
```
do_execsql_test pragma-cache-size {
  PRAGMA page_count
} {2}
```

Reply:
I won't be able to do this in the pragma.test. This is because that test is ran with two different databases as an input (testing.db and testing_norowidalias), and they each have a different page count.

I will see if I can come up with a simpler unit test somewhere.